src/components/FullDetailView/FullDetailPage.js

```diff
@@ -76,29 +79,29 @@ export class FullDetailPage extends Component {
 
   // Get the data for the current definition
   handleNewSpec(component) {
-    const { token, uiInspectGetDefinition, uiInspectGetCuration, uiInspectGetHarvested } = this.props
+    const { session, uiInspectGetDefinition, uiInspectGetCuration, uiInspectGetHarvested } = this.props
     if (!component) return
-    uiInspectGetDefinition(token, component)
-    uiInspectGetCuration(token, component)
-    uiInspectGetHarvested(token, component)
+    uiInspectGetDefinition(session.token, component)
```

Comment:
Yes. After adding the username to the state session, we were mapping two `session` properties in a few component Props.
I thought it would reduce the number of props, but I did this in a separate commit so I can revert it if you think it doesn't add much value

Reply:
As said in our voice conversation, i think it's a much better approach to have both `session` and `token` props, in order to avoid any refactoring on other implementations.